### PR TITLE
Compare files manually.

### PR DESCRIPTION
**Pull request by jerjou**
Addresses #707

### CODE REVIEW

**Comment by jonparrott** [comment]
Hmm, I liked the idea of just closing the handle to the named temporary file and then using filecmp.

**Comment by jerjou** [comment]
Oh. It just seemed less clean. eg:
* You're now manually in charge of file cleanup, which feels like it should be the context manager's job
* The use of `filecmp` with a filename when you've already got a handle on the file always felt a bit hacky.
* The file comparison is now both inside and outside of the context manager, which breaks it up visually even though it's the same conceptual operation

Granted, this is a bit more verbose and low-level, but I feel like it flows better.

**Comment by jonparrott** [comment]
Can we just remove the file comparison altogether from this sample?

**Comment by jerjou** [comment]
Oh yeah - sure. Apparently this file isn't even referenced in the docs anymore.
Done.

File: storage/api/crud_object.py [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
 import argparse
-import filecmp
 import json
 import tempfile
 
 from googleapiclient import discovery
 from googleapiclient import http
 
 from oauth2client.client import GoogleCredentials
+from six.moves import zip_longest
```
Comment (jerjou) [inline storage/api/crud_object.py]: Ah. right.

**Pull request merged**